test/integrate/dubbo/go-server/go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/apache/dubbo-go/test/integrate/dubbo/go-server
 
 go 1.13
+
+require (
+	github.com/apache/dubbo-go v1.5.5
```

Comment:
remove this block pls，it will cause some problems when run integration test in Github Action.